### PR TITLE
Update instance properties when create/update/delete data element

### PR DESCRIPTION
**Pull request by HenningNormann**
- Implemented PgInstanceRepository.SetStatuses in the db functions/procs for data element create/update/delete
- Updated dataelement.LastChangedBy in the Delete method in DataController
- Added lastChangedBy and isRead to data element test data
- Changed references to appsettings.json in test artifacts  to include path
- Added readstatus to testdata Instance_1_1

### CODE REVIEW

File: src/Storage/Controllers/DataController.cs [hunk 1]
```diff
@@ -130,6 +130,7 @@ public async Task<ActionResult<DataElement>> Delete(int instanceOwnerPartyId, Gu
                 return await InitiateDelayedDelete(instance, dataElement);
             }
 
+            dataElement.LastChangedBy = User.GetUserOrOrgId();
```
Comment (acn-sbuad) [inline src/Storage/Controllers/DataController.cs:133]: should we also set it in the case of delayed delete?

File: test/UnitTest/data/cosmoscollections/dataelements/11f7c994-6681-47a1-9626-fcf6c27308a5.json [hunk 2]
```diff
@@ -9,5 +9,7 @@
   "refs": [
   ],
   "created": "2020-05-11T17:09:28.4621953Z",
-  "lastChanged": "2020-05-11T17:09:28.4621953Z"
+  "lastChanged": "2020-05-11T17:09:28.4621953Z",
```
Comment (acn-sbuad) [inline test/UnitTest/data/cosmoscollections/dataelements/11f7c994-6681-47a1-9626-fcf6c27308a5.json:12]: Is it still relevant to update a cosmoscollection of testdata? Do we need to rename the folder?

File: src/Storage/Controllers/DataController.cs (hunk 1 above)
Comment (HenningNormann) [reply src/Storage/Controllers/DataController.cs:134]: Done

**Review by acn-sbuad — APPROVED** [review]
🎈

**Pull request merged**